### DISCUSSION TITLE
[Q]: Tags or lists. What do you use ?

### DISCUSSION THREAD

**Discussion by Boss-man** [karakeep-app/karakeep#738] [General]
Hi

Just got Hoarder installed and testing things out. Currently using Readeck, but really like the mobile app for hoarder.
In Hoarder I can use tags or lists, but only seems to be able to use lists in the Android app.
In Readeck I used labels, which are the same as tags, I guess.

For example I have tags as "Blue Iris", "Home Assistant", "Proxmox", "Work", "SCCM" and so on.
I could just as well have lists for that instead.

How do you organize your links ?  Tags or lists. Or both ?

**Comment by MohamedBassem** [comment]
@Boss-man personally, the way I use it is that tags are mostly the AI generated ones, and lists are for what I want to sort out manually.

**Reply by Boss-man** [reply]
Thanks, I'll try to switch over to using lists.

Is there a way to see items that are NOT in a list ?

**Reply by MohamedBassem** [reply]
@Boss-man not yet, but it's planned
